### ISSUE TITLE
recovery_loss = tf.reduce_sum(tf.pow(filtered_image_batch - input_data_clean, 2.0))

### ISSUE THREAD

**Issue by Unclechan97** [wenyyu/Image-Adaptive-YOLO#72]
首先感谢你为我们提供的新思路，然后我想问一下，代码中的recovery_loss有什么作用呢？训练时recovery loss没有降低，而且为什么要将输入图像和处理后的图像作差呢？
希望能得到你的解答，感谢！

**Comment by andwizard** [comment]
recovery loss 都没有参与训练，作者应该只是单纯的算出来展示一下
